tools/coq_makefile.ml

```diff
@@ -151,6 +151,51 @@ let generate_makefile oc conf_file local_file local_late_file dep_file args proj
     ] in
   output_string oc s
 
+let generate_meta_file p =
+  try
+    match p.meta_file with
+    | Absent -> p
+    | Generate proj ->
+        let cmname = List.map (fun { thing } -> thing) (p.mllib_files @ p.mlpack_files) in
+        let dir, cmname =
+          match cmname with
+          | [] -> Printf.eprintf "In order to generate a META file one needs an .mlpack or .mllib file"; exit 1
+          | [x] -> Filename.dirname x, Filename.(basename @@ chop_extension x)
+          | _ -> Printf.eprintf "Automatic META generation only works for one .mlpack or .mllib file, since you have more you need to write the META file by hand"; exit 1 in
```

Comment:
```suggestion
          | [] -> Printf.eprintf "In order to generate a META file one needs an .mlpack or .mllib file\n"; exit 1
          | [x] -> Filename.dirname x, Filename.(basename @@ chop_extension x)
          | _ -> Printf.eprintf "Automatic META generation only works for one .mlpack or .mllib file, since you have more you need to write the META file by hand\n"; exit 1 in
```